src/components/SignIn/SignIn.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { Button, Checkbox, Form, Icon, Input, Row } from "antd";
+import openNotification from "../ReusableComponents/Notification";
+import { PASSWORD_POLICY } from "../../constants/validation";
+import { Link } from "react-router-dom";
+
+const formLayout = {
+  wrapperCol: {
+    sm: { span: 16, offset: 4 },
+    md: { span: 12, offset: 6 },
+    lg: { span: 8, offset: 8 },
+    xl: { span: 6, offset: 9 },
+    xxl: { span: 4, offset: 10 }
+  }
+};
+
+class SignIn extends React.Component {
+  handleSubmit = e => {
+    e.preventDefault();
+    const { validateFields } = this.props.form;
+
+    const onSuccess = values => {
+      console.log(values);
+    };
+
+    validateFields((err, values) => {
+      err
+        ? openNotification({
+            type: "warning",
+            message: "Заполните все обязательные поля",
+            description: "Поля со звездочкой * обязательны для заполнения."
+          })
+        : onSuccess(values);
+    });
+  };
+
+  render() {
+    const { getFieldDecorator } = this.props.form;
+
+    return (
+      <Form onSubmit={this.handleSubmit} {...formLayout}>
+        <Form.Item>
+          {getFieldDecorator("email", {
+            rules: [{ required: true, message: "Пожалуйста введите Ваш email" }]
+          })(
+            <Input
+              prefix={<Icon type="mail" style={{ color: "rgba(0,0,0,.25)" }} />}
```

Comment:
move to reusable styled component

Reply:
After the changes, I have an error below: 
![image](https://user-images.githubusercontent.com/19416734/72366219-8049bd80-3702-11ea-89b6-e1424a5d63d6.png)

Reply:
After the Icon removing, I have the same error
![image](https://user-images.githubusercontent.com/19416734/72366365-cbfc6700-3702-11ea-9c7e-0761ce935529.png)

Reply:
Should I resolve this problem or just pass styles as props?